### PR TITLE
Add Hubspot

### PR DESCRIPTION
**Pull request by jychen7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,22 +30,23 @@ Curious to know how big companies are operating their kafka fleet in production?
...
-25. [Shopify](#shopify)
-26. [Slack](#slack)
-27. [Stripe](#stripe)
-28. [Uber](#uber)
-29. [Wise](#wise)
-30. [Wix](#wix)
-31. [Yelp](#yelp)
-32. [Zalando](#zalando)
-33. [Zopa Bank](#zopa-bank)
+18. [Hubspot](#hubspot)
```
Comment (jychen7) [inline README.md:33]: [unrelated] propose switching all this to `1.`, so new PR generates fewer changes. Markdown will automatically create the sequence anyway

Comment (dttung2905) [reply README.md:33]: @jychen7 Sure. I think it makes sense. Could you help to change it in your PR as well?

**Review by dttung2905 — COMMENTED** [review]
LGTM!. Just a minor comment for me and we are good to go!

File: README.md (hunk 1 above)
Comment (jychen7) [reply README.md]: sure, 156b9fe23c74e6dee5c80b859c666aa298d1862c

**Review by dttung2905 — APPROVED** [review]
LGTM! Thanks again for your contribution and suggestion

**Pull request merged**